apps/web/playwright/managed-event-types.e2e.ts

```diff
@@ -1,212 +1,247 @@
+    const adminUserSnapshot = await adminUser.self();
+    await apiLogin({ ...adminUserSnapshot, password: adminUserSnapshot.username }, adminPage);
+    await adminPage.goto("/event-types");
+    await adminPage.getByTestId("event-types").locator(`a[title="${teamEventTitle}"]`).click();
+    await adminPage.waitForURL("event-types/**");
+    await adminPage.locator('input[name="length"]').fill(`45`);
+    await submitAndWaitForResponse(adminPage);
+    await adminContext.close();
+
+    await memberPage.goto("/event-types");
+    await memberPage.getByTestId("event-types").locator('a[title="Managed Event Title"]').click();
+    await memberPage.waitForURL("event-types/**");
+    //match length
+    expect(await memberPage.locator("[data-testid=duration]").getAttribute("value")).toBe("45");
+    //ensure description didn't update
+    expect(await memberPage.locator(`input[name="title"]`).getAttribute("value")).toBe(`Managed Event Title`);
+    await memberPage.locator('input[name="title"]').fill(`managed`);
+    // Save changes
+    await submitAndWaitForResponse(memberPage);
+  });
 
-    await test.step("Managed event type should only update the unlocked fields modified by Admin", async () => {
-      await memberUser.apiLogin();
-      await page.goto("/event-types");
-
-      await page.getByTestId("event-types").locator('a[title="managed"]').click();
-      await page.waitForURL("event-types/**");
-
-      await expect(page.locator('input[name="title"]')).toBeEditable();
-      await page.locator('input[name="title"]').fill(`Managed Event Title`);
-      // Save changes
-      await page.locator('[type="submit"]').click();
-      await page.getByTestId("toast-success").waitFor();
-      await page.waitForLoadState("networkidle");
-
-      await page.goto("/auth/logout");
-
+  const MANAGED_EVENT_TABS: { slug: string; locator: (page: Page) => Locator | Promise<Locator> }[] = [
+    { slug: "setup", locator: (page) => getByKey(page, "title") },
+    {
+      slug: "team",
+      locator: (page) => getByKey(page, "automatically_add_all_team_members"),
+    },
+    {
+      slug: "availability",
+      locator: (page) => getByKey(page, "members_default_schedule_description"),
+    },
+    {
+      slug: "limits",
+      locator: (page) => getByKey(page, "before_event"),
+    },
+    {
+      slug: "advanced",
+      locator: (page) => getByKey(page, "event_name_in_calendar"),
+    },
+    {
+      slug: "apps",
+      locator: (page) => page.getByRole("heading", { name: "No apps installed" }),
+    },
+    {
+      slug: "workflows",
+      locator: (page) => page.getByTestId("empty-screen").getByRole("heading", { name: "Workflows" }),
+    },
+    {
+      slug: "ai",
+      locator: (page) => page.getByTestId("empty-screen").getByRole("heading", { name: "Cal.ai" }),
+    },
+  ];
+
+  MANAGED_EVENT_TABS.forEach((tab) => {
+    test(`Managed event can render "${tab.slug}" tab`, async ({ page, users }) => {
```

Comment:
This would have helped catching a previous error where the availability tab stopped rendering properly.